Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1118,14 +1136,23 @@ static bool isCopyDisplayedPixelBufferAvailable()
             return;
         weakThis->didProvideMediaDataForTrackId(WTFMove(sample), trackId, mediaType);
     });
-    m_parser->setCallOnClientThreadCallback([](auto&& function) {
-        function();
-    });
```

Comment:
This used to force the parser to make its callbacks on whatever thread it happened to be on, we need to remove this so callbacks happen on the main thread as expected.